routes/projects/update.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * PUT /projects/:username/:projectName
+ * Creates a new project for the given user. A project object is sent in
+ * the form:
+ * {
+ *   name: <string> required,
+ *   username: <string> required,
+ *   isPublic: <boolean> required,
+ *   publishedAt: <dataTime> optional,
+ *   publishLink: <string> optional,
+ *   files: [ <FileObj1>, <FileObj2>... ] required
```

Comment:
So what's the verdict on this?
POST /projects body:
```js
{
  ...
  files: []
}
```
PUT /projects/:username/:prjName
```
same as above
```
GET /projects/:username/:prjName
```
{
  ...
  files: [ 
            { 
               ..., 
               link: linkTofile 
            }, 
            ... 
         ]
}
```

Is that it?
@ashleygwilliams @humphd @sedge